src/core_plugins/testbed/public/testbed.html

```diff
@@ -4,7 +4,37 @@
     <!-- All markup within the CONTENT section below can be deleted and replaced with whatever markup
     you need to demonstate your functionality. Nothing need be preserved. -->
     <!-- CONTENT START -->
+    <table class="kuiTable kuiVerticalRhythm">
+      <thead>
+        <tr class="kuiTableRow">
+          <th class="kuiTableHeaderCell kuiTableHeaderCell--checkBox"></th>
+          <th class="kuiTableHeaderCell">Item</th>
+        </tr>
+      </thead>
+      <tr
+        ng-repeat="item in testbed.myItems"
+        class="kuiTableRow"
+      >
+        <td class="kuiTableRowCell kuiTableRowCell--checkBox">
+          <div class="kuiTableRowCell__liner">
+            <selectable-row
+              id="item.id"
+              is-selected="item.isSelected"
+              on-select-change="testbed.onSelectChange"
+            ></selectable-row>
```

Comment:
Yeah, I'm good with your proposal. It also eliminates the need for another component that I was about to promote into OSS Kibana: `selectable-column` which is used for a "select all" functionality. It makes sense to me that this should be a generic component that could be used in various contexts.

I'm +1 for calling this `check-box` (looking through other Angular directives in Kibana, we don't have the `kui` prefix convention anywhere so I don't want to introduce it now). 

@BigFunger Your thoughts before I make this broad change affecting multiple places in this PR?